docs/README.md

```diff
@@ -0,0 +1,212 @@
+
+**Getting desired capabilities**
+1. Go to https://portal.kobiton.com/login and login with your Kobiton account.
+2. Click **"Devices"** at the top of the window.
+
+![Devices button](./assets/devices-button.png)
+
+3. Hover over the device you want to run the test script on (in this example: `Pixel 2 XL`), click the gear button.
+
+![Automation Settings button](./assets/gear-button.png)
+
+4. In the `Automation Settings` popup:
+- In `Language` section, choose `NodeJS`.
+- In `App Type` section, choose `Hybrid/Native from Url`.
+- In `Application Url` section, put `https://appium.github.io/appium/assets/ApiDemos-debug.apk`.
+
+![Example Android DesiredCaps](./assets/desired-caps-example-android.png)
+
+**Setting desired capabilities**
+
+Add these environment variables to your project configuration in CircleCI with values of corresponding variables taken from the collected desired capabilities as shown in the table below
+
+| CircleCI Environment Variable       | Desired Capabilities Variable  | Description                                         | Default Value                                             |
+|-------------------------------------|--------------------------------|-----------------------------------------------------|-----------------------------------------------------------|
+| KOBITON_DEVICE_PLATFORM_NAME        | platformName                   | Kobiton Device Platform Name (e.g: Android, iOS)    | Android                                                   |
+| KOBITON_DEVICE_NAME                 | deviceName                     | Kobiton Device Name                                 | Galaxy*                                                   |
+| KOBITON_DEVICE_PLATFORM_VERSION     | platformVersion                | Kobiton Device Platform Version                     | *No*                                                      |
+| KOBITON_SESSION_DEVICE_ORIENTATION  | deviceOrientation              | Device Orientation (e.g: Portrait, Landscape)       | portrait                                                  |
+| KOBITON_SESSION_CAPTURE_SCREENSHOTS | captureScreenshots             | Enable screenshots capture during session execution | true                                                      |
+| KOBITON_SESSION_DEVICE_GROUP        | deviceGroup                    | Kobiton device group                                | KOBITON                                                   |
+| KOBITON_ORGANIZATION_GROUP_ID       | groupId                        | Group ID of the device                              | *No*                                                      |
+| KOBITON_SESSION_APPLICATION_URL     | app                            | Download link to the application used for testing   | https://appium.github.io/appium/assets/ApiDemos-debug.apk |
+
+For example, if desired capabilities for executing automation test of the provided demo Android application on Pixel 2 XL running Android 8.1.0:
```

Comment:
move this to the top of this part

Reply:
I think we should keep this here. Flow : show instructions -> demonstration (example)